runtime/opt/taupage/init.d/86-adjust-application-logrotate

```diff
@@ -30,6 +30,18 @@ if [ -n "$config_application_logrotate_rotate" ]; then
         sed -i "s/\brotate\b.*$/rotate $config_application_logrotate_rotate/g" $application_logrotate
 fi
 
+#disable the copytruncate and use the default behavior for logrotate
+if [ -n "$config_application_logrotate_disable_copytruncate" = True ]; then
```

Comment:
If I don't use `True` it would be also disabled with `application_logrotate_disable_copytruncate: false`